src/index.html

```diff
@@ -11,9 +11,25 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
 </head>
 <body>
-
+  <div class="block" data-qa="card">
+    <img src="images/imac.jpeg" alt="computer" class="image"/>
+    <p class="name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+    <p class="code">Код товара: 195434</p>
+    <div class="stars">
+      <img src="images/star-active.svg" alt="star-active" class="star-active"/>
+      <img src="images/star-active.svg" alt="star-active" class="star-active"/>
+      <img src="images/star-active.svg" alt="star-active" class="star-active"/>
+      <img src="images/star-active.svg" alt="star-active" class="star-active"/>
+      <img src="images/star.svg" alt="star" class="star"/></p>
+      <span class="fee">Отзывов: 5</span>
+    </div>
+    <div class="prise">Цена:</div>
```

Comment:
create common div for price and change div => p
```suggestion
    <p class="prise">Цена:</p>
```